bots/images/scripts/fedora.setup

```diff
@@ -27,16 +29,21 @@ if [ $(uname -m) = x86_64 ]; then
     HAVE_KUBERNETES=1
 fi
 
+HAVE_DOCKER=
+if type docker >/dev/null 2>&1; then
+    HAVE_DOCKER=1
+fi
```

Comment:
Wait, that looks wrong now -- this test is done before the docker package gets installed below. As this is chicken & egg, let's go back to testing `if [ "$1" = fedora-31 ]`.  Sorry for the bogus advice before!